user-guides/data-farming/how-to-challengedf.md

```diff
@@ -0,0 +1,60 @@
+---
+description: Make extra dosh with building prediction models
+---
+
+# How to Challenge DF
+
+<figure><img src="../../.gitbook/assets/rewards/gif/challenge-df.gif" alt=""></figure>
+
+## Predict to earn
+
+If you're eager to begin extracting value from data but aren't sure how to get started, then [Challenge DF is here to help](../../rewards/df-challengedf.md)!
+
+By participating in the weekly data challenges of predicting the price of Ethereum, you will become familiar with the Ocean Protocol stack and with building predictions that can be sold for money.
+
+These are the steps to get involved:
+
+#### Step 1 - Navigate to the Data Farming dApp
+
+Go to https://df.oceandao.org
+
+#### Step 2 - Go to the "Challenges" page
+
+After the dApp loads, look for the menu and click on "Challenges"
+
+<figure><img src="../../.gitbook/assets/rewards/challenge-page.png" alt=""><figcaption><p>Find "Challenges" in the menu and click on it</p></figcaption></figure>
+
+#### Step 3 - Click on "Participate"
+
+After clicking on "Participate" you will be [linked to the Challenge DF README](https://github.com/oceanprotocol/predict-eth/blob/main/challenges/challenge-df.md) that outlines all details about the challenge.
+
+<figure><img src="../../.gitbook/assets/rewards/challenge-active.png" alt=""><figcaption><p>Click the Participate button</p></figcaption></figure>
+
+#### Step 4 - Use README to publish your predictions
+
+You can see the ongoing challenge right away together with some key informations such as submission deadline and available rewards.
```

Comment:
Why do I need to do the previous steps when all it's happening is  I'm redirected to GitHub the README?

Reply:
The readme it's just a step in the user flow which mainly focuses on technical detail about how to submit the predictions. We described there the whole user flow from the DF and active challenges to how to claim rewards and see the winners

Reply:
The user-guides and you're reading have to do with Data Farming & Challenge DF. We're onboarding you onto the high level.

```
Why do I need to do the previous steps when all it's happening is that I'm redirected to GitHub the README?
```

The links you're being redirected to, are the nitty-gritty details and the technical document that needs to be updated whenever the system changes.

This is not meant to exist in the docs.